.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: node_js
 node_js:
+  - "4.0"
```

Comment:
No, fair enough. It's none of my business tbh, you're the ones using the project after all. :grin: